ynab/ynab.py

```diff
@@ -108,12 +109,17 @@ def from_json(cls, data):
         return instance
 
     def _init_data(self, data):
-        self.precision = 2
+        # TODO:
+        # - scheduledTransactions
+        # - accountMappings
+        # - monthlyBudgets
         self._accounts = Accounts._from_flat(self, data['accounts'])
         self._payees = Payees._from_flat(self, data['payees'])
         self._master_categories = MasterCategories._from_flat(self, data['masterCategories'])
         self._transactions = Transactions._from_flat(self, data['transactions'])
         self._transactions.sort_by('date')
+        self._meta_data = BudgetMetaData._from_flat(self, data['budgetMetaData'])
```

Comment:
There's a precedent with `_master_categories` above, so not sure about that, I was just following the capitalization of the field from the JSON. But if you insist, I'll rename it.